src/engines/Cesium/hooks/useEngineRef.ts

```diff
@@ -983,6 +988,49 @@ export default function useEngineRef(
         if (!viewer || viewer.isDestroyed()) return;
         return getCredits(viewer);
       },
+      setGeoidServer: (geoidServerParams: GeoidServer) => {
```

Comment:
Do you think it's possible to set it through props, maybe [GlobeProperty](https://github.com/reearth/core/blob/alpha/src/Map/types/viewerProperty.ts#L52)?